test/performance/config/local.env

```diff
@@ -0,0 +1,8 @@
+C9_BACKEND_HOST="cloud-nine-wallet-backend"
+HLB_BACKEND_HOST="happy-life-bank-backend"
+C9_AUTH_HOST="cloud-nine-wallet-backend"
```

Comment:
Isn't the auth host here suppose to be `(cloud-nine-wallet/happy-life-bank)-auth`?

Reply:
Yes, fixed. Thanks.